### ISSUE TITLE
source 4.30 failed to build on linux/armv7

### ISSUE THREAD

**Issue by akuropka** [SoftEtherVPN/SoftEtherVPN#936]
Applies to: v4.30-9695/9696 (worked before)
System: Raspberry PI 3B+ ArchLinux ARM

Makefile:
[Makefile.txt](https://github.com/SoftEtherVPN/SoftEtherVPN/files/3367750/Makefile.txt)

Checking out the source, confugure and building fails:
```
cc -DNDEBUG -DVPN_SPEED -DUNIX -DUNIX_LINUX -D_REENTRANT -DREENTRANT -D_THREAD_SAFE -D_THREADSAFE -DTHREAD_SAFE -DTHREADSAFE -D_FILE_OFFSET_BITS=64 -I./src/ -I./src/Cedar/ -I./src/Mayaqua/ -O2 -fsigned-char -c src/Mayaqua/Network.c -o tmp/objs/Mayaqua/Network.o
src/Mayaqua/Network.c: In Funktion »SslCertVerifyCallback«:
src/Mayaqua/Network.c:5974:11: Fehler: Dereferenzierung eines Zeigers auf unvollständigen Typen »X509_STORE_CTX« {alias »struct x509_store_ctx_st«}
    if (ctx->cert != NULL)
           ^~
make: *** [Makefile:135: tmp/objs/Mayaqua/Network.o] Fehler 1
```

The linux-arm_eabi-32bit-precompiled packages do work though.

**Comment by chipitsine** [comment]
are you using "do deprecated" openssl build ?

please see https://github.com/SoftEtherVPN/SoftEtherVPN/issues/864

also, this repo is about "SE VPN Developer edition" which is 5.0.1
I'm afraid it is wrong place for 4.30 questions

**Comment by akuropka** [comment]
Oh, I am sorry (since there was no place to report issues...).
It works on 5.0.1 with same configuration (openssl library).

**Comment by metalefty** [comment]
@akuropka FYI, the issue has been fixed at v4.30-9700-beta. 
See also: https://github.com/SoftEtherVPN/SoftEtherVPN_Stable/pull/5
